examples/jwt/README.md

```diff
@@ -0,0 +1,43 @@
+# JWT Authentication
+
+Example that showcase the use of jwt interceptors using Xiana.
+
+
+## Run the backend
+
+```bash 
+lein run
+```
+
+## Run manual tests against the application
+
+In this example, 3 endpoints are provided:
+
+#### localhost:3000/login
+
+This is the only unprotected endpoint, meaning this endpoint will require no authorization header with a jwt token. This endpoint is the one that will provide you with the token to be put in the header.
+
+Make a request to this endpoint by running the following:
+```bash
+curl -d '{"password":"topsecret", "email":"xiana@test.com"}' -H "Content-Type: application/json" -X POST http://localhost:3000/login
+```
+
```

Comment:
This gives me "Missing credentials"

Reply:
@soulflyer You're right. 
@a-luz The `public-key` and the `private-key` variables are read from environment, right?